### PR TITLE
Add support for BROKER_TRANSPORT_OPTIONS

### PR DESCRIPTION
**Pull request by CasperCL**
Add support for BROKER_TRANSPORT_OPTIONS in Celery 4.1 and up.

### CODE REVIEW

**Comment by CasperCL** [comment]
Fixes #67

**Comment by sontek** [comment]
Thanks!  I'll get this merged and released today

**Comment by noelwilson-qualio** [comment]
> Thanks! I'll get this merged and released today

This would be very helpful thank you :)

**Comment by sontek** [comment]
@CasperCL Can you  merge master into this PR so we can see the tests passing?   Then I think we are good to go

File: pyramid_celery/loaders.py [hunk 1]
```diff
@@ -106,6 +106,13 @@ def read_configuration(self, fail_silently=True):
             if setting in config_dict:
                 split_setting = config_dict[setting].split()
                 config_dict[setting] = split_setting
+                
+        dict_settings = ['BROKER_TRANSPORT_OPTIONS']
```
Comment (sontek) [inline pyramid_celery/loaders.py:110]: Celery is deprecating the ALL CAPS options.  Lets name this `broker_transport_options`.  I've went through and cleaned up the rest of the options that were mostly for celery 3 and earlier

**Comment by CasperCL** [comment]
Oh I didn't realise the build was fixed on master! My repo is now up to date with yours.

File: pyramid_celery/loaders.py (hunk 1 above)
Comment (CasperCL) [reply pyramid_celery/loaders.py]: Sounds good! It's done.

**Comment by sontek** [comment]
@CasperCL I'm thinking of updating this API a bit.  You've designed it out to look like this:

```
broker_transport_options = {'visibility_timeout': 18000}
```

Which could get weird to work with / read because you can't use whitespace or anything in it.  I'm thinking instead that we put the key/value pairs underneath:

```
broker_transport_options =
    visibility_timeout: 18000
    max_retries: 5

```

or introducing a new configuration section which would be more ini like:

```
[celery:broker_transport_options]
visibility_timeout = 18000
max_retries = 5
```

I'm leaning towards the second for semantics of the ini file but it'd be interesting to hear your thoughts

**Comment by sontek** [comment]
Actually, I just realized we do the second way for both celerybeat and celeryroute, so I think thats the way to go

**Comment by CasperCL** [comment]
@sontek I'm all for it. I won't have time to change this until the weekend, so you're probably better off to do it yourself if it's holding you back.

**Comment by sontek** [comment]
@CasperCL How does this look?

https://github.com/sontek/pyramid_celery/pull/86

**Comment by sontek** [comment]
Implemented in #86

**Pull request closed without merging**